packages/@glimmer/runtime/lib/vm/arguments.ts

```diff
@@ -169,8 +169,8 @@ export class PositionalArgumentsImpl implements PositionalArguments {
     return check(stack.get(position, base), CheckPathReference) as T;
   }
 
-  capture(): CapturedPositionalArgumentsImpl {
-    return new CapturedPositionalArgumentsImpl(this.references);
+  capture(): CapturedPositionalArguments {
+    return this.references as CapturedPositionalArguments;
```

Comment:
`CapturedPositionalArgs, `CapturedNamedArgs`, and `CapturedArgs` all have a fake symbol as part of the interface. The idea being, you must use one of these APIs to make them, you can't just make a dictionary and use it as arguments. We don't want to actually make or assign the symbol because that would be costly, so we use `as` to typecast it in the creation APIs.